docs/blog/latex-output-tables/index.qmd

```diff
@@ -0,0 +1,286 @@
+
+In this post, we will provide examples of generating LaTeX outputs of tables using the `as_latex()` method, which is new in Great Tables `v0.14.0`. We'll go over some LaTeX workflows in Quarto and the current options for Great Tables within that publishing system. Then, we will address some of the caveats and current limitations for exporting tables to LaTeX while providing an outlook on how those will be addressed in the future.
+
+## Generating a LaTeX table with Great Tables
+
+When making a LaTeX output table with Great Tables, the code shouldn't be that much different from a table meant for HTML output. While HTML tables are rendered automatically in notebooks (as GT objects come with an HTML representation), getting the table as LaTeX requires the use of the new `as_latex()` method. Here is an example where a basic table is generated, and the LaTeX table code is returned and printed.
+
+```{python}
+from great_tables import GT
+from great_tables.data import gtcars
+import polars as pl
+
+gtcars_pl = (
+    pl.from_pandas(gtcars)
+    .select(["mfr", "model", "hp", "trq", "mpg_c", "msrp"])
+)
+
+gt_tbl = (
+    GT(
+        gtcars[["mfr", "model", "hp", "trq", "msrp"]].head(5),
+    )
+    .tab_header(
+        title="Some Cars from the gtcars Dataset",
+        subtitle="Five Cars are shown here"
+    )
+    .tab_spanner(
+        label="Make and Model",
+        columns=["mfr", "model"],
+        id="make_model"
+    )
+    .tab_spanner(
+        label="Performance",
+        columns=["hp", "trq", "msrp"]
+    )
+    .tab_spanner(
+        label="Everything but the cost",
+        columns=["mfr", "model", "hp", "trq"]
+    )
+    .fmt_integer(columns=["hp", "trq"])
+    .fmt_currency(columns="msrp")
+    .tab_source_note("Cars are all 2015 models.")
+    .tab_source_note("Horsepower and Torque values are estimates.")
+)
+
+print(gt_tbl.as_latex())
+```
+
+The table example shows how some of Great Tables' structural components are represented in LaTeX. We can add a title to the table with `tab_header()` and it will be placed in a caption (where the title and subtitle will be sized appropriately). Spanners, which are something of a pain to manually author, work well here, and we can use an unlimited number of spanner levels. Number formatting is handled correctly here with `fmt_integer()` and `fmt_currency()` (notice that `$` is escaped as `\$`). Finally, source notes are placed below the table in a minipage environment.
+
+A frequent bugbear with any programmatic generation of LaTeX tables is LaTeX escaping. Not escaping correctly often leads to LaTeX rendering errors and this can break an entire publishing workflow. Thankfully, Great Tables will automatically and correctly escape characters in LaTeX and this will happen with:
```

Comment:
I think we could cut some of this down (e.g. to say in a short sentence or two that Great Tables escapes where necessary, etc..)

Reply:
The escaping paragraph is now much shorter.